Cargo.toml

```diff
@@ -18,6 +18,7 @@ minimp3_fixed = { version = "0.5.4", optional = true}
 symphonia = { version = "0.5.4", optional = true, default-features = false }
 crossbeam-channel = { version = "0.5.8", optional = true }
 thiserror = "1.0.49"
+assert_float_eq = "1.1.3"
```

Comment:
can we move this to a dev-dependency? since its only needed for the test. 

assert_float_eq might be somewhat outdated. Some alternatives:
- https://crates.io/crates/approx
- https://crates.io/crates/float-cmp

Or you could always drop the dependency and use 
```rust
(faded.next().unwrap() - 0.5).abs() < 0.0001
```
though thats a bit less readable...

Reply:
Now using approx in f2f93e6